tests/test_config/test_config_schema.py

```
@@ -106,6 +106,75 @@ def test_minimal_api_config(self):
         assert cfg.benchmarks[0].name == "gsm8k"
         assert isinstance(cfg.benchmarks[0].solver, SimpleSolver)
         assert cfg.benchmarks[0].solver.service == "solver"
+        assert cfg.output.trajectories.enrich is True
+
+    @pytest.mark.parametrize(
+        "service_overrides,output_overrides,expected_capture,expected_enrich",
```

Ruff (0.15.18)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)